tasks/update.rake

```diff
@@ -2,12 +2,16 @@ require_relative 'lib/slack_api/methods_spider'
 require_relative 'lib/slack_api/events_spider'
 require_relative 'lib/slack_api/spec_validator'
 
+def delete_files_except_undocumented(*dirs)
```

Comment:
As a rule of thumb you should put these in a module otherwise it pollutes global namespace. NBD

Reply:
Yeah that was my initial idea as well but it felt like overkill for a single two-line method...

Reply:
But we're better than this! I think a better way to fix this is to add an actual Rake task that deletes existing files and takes a parameter for the folder(s), then you can `invoke` it here, much cleaner than a random method.